test/Sentry.Maui.Device.TestApp/Sentry.Maui.Device.TestApp.csproj

```diff
@@ -1,12 +1,13 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks></TargetFrameworks>
-    <TargetFrameworks Condition="'$(NO_ANDROID)' == ''">$(TargetFrameworks);net8.0-android;net9.0-android</TargetFrameworks>
-    <TargetFrameworks Condition="'$(NO_IOS)' == '' And $([MSBuild]::IsOSPlatform('OSX'))">$(TargetFrameworks);net8.0-ios;net9.0-ios</TargetFrameworks>
+<!--    <TargetFrameworks>$(TargetFrameworks);net8.0-android</TargetFrameworks>-->
```

Comment:
I think we want to test both net8.0 and net9.0 targets. 

At the moment we only run device tests for both tfms for Android, which I added in [this PR](https://github.com/getsentry/sentry-dotnet/commit/13d4a9f2038e142f9dc8c7799df06edfe17a2dd4#diff-c709796952b06c9c4ae93d17583aacd1d473093651a6de9b275ce15a2ddca0f8). It was necessary to test/implement that functionality but I didn't want the PR to bloat out to 2000 lines so I left iOS out of it. 

We should really setup the iOS device tests to run in a matrix in the same way though (rather than arbitrarily testing against one tfm or the other).

Reply:
I actually think we should remove net8 for several reasons
- dotnet android & ios are out of support on net8 already
- It takes a ton of extra time to build those additional targets
- It was really causing grief for the visual runner which I would really REALLY like to use going forward

Thoughts?

Reply:
Hm, tricky to remove for Android. We'd have no way to test the older `AssemblyReader` (unless we drop support for that as well).

Could we conditionally run the visual tests only on net9.0 (to avoid whatever issues we're having with these in net8.0)?

Is so then maybe we could drop support for net8.0 in the next major release (it seems like a pretty major change to the SDK).

Reply:
I don't know how we're using a global.json where we're multitargeting different major sdks and workload sets that are tied to a specific major version.  I don't know how our device tests are even running at the moment

Reply:
I've brought over an improved setup I did in another pending PR for gesture recognizers.  This should hopefully fix all of this stuff up for us and makes it easy to bring in the visual runner when we need it.